### ISSUE TITLE
Add a LICENSE

### ISSUE THREAD

**Issue by akselaase** [bramvbilsen/Matlab-Code-Run-for-VS-Code#14]
The project is missing a license statement, which makes it hard to fork and continue development. Creating derivative works isn't permitted without you explicitly allowing it. It would be great if you could clarify the situation by determining a license for your project, thanks.

**Comment by mauzigoe** [comment]
Yeah, this is needed. Without a License one has to assume that all rights are reserved :/
